chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt

```diff
@@ -0,0 +1,153 @@
+package com.droidconke.chai.typography
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ProvidableCompositionLocal
+import androidx.compose.runtime.compositionLocalOf
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.LineHeightStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.em
+import androidx.compose.ui.unit.sp
+import com.droidconke.chai.ChaiTheme
+import com.droidconke.chai.atoms.MontserratSans
+import com.droidconke.chai.utils.ChaiPreview
+
+private val DUMMY_CHAI_TEXT = "Design systems provide guidelines and reusable components to ensure consistent and efficient design across digital products, improving user experience and streamlining team workflows."
+

```

Comment:
Would it make sense to use  an inline doc here... showing how we use the specific composable